x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityTests.java

```diff
@@ -619,7 +619,13 @@ public void testValidateForFipsNoErrorsOrLogs() throws IllegalAccessException {
                 )
             )
             .put(
-                XPackSettings.PASSWORD_HASHING_ALGORITHM.getKey(),
+                XPackSettings.SERVICE_TOKEN_HASHING_ALGORITHM.getKey(),
```

Comment:
The diff looks weird because there was a duplicate entry for `XPackSettings.PASSWORD_HASHING_ALGORITHM.getKey()` here before